src/main/java/io/vertx/ext/amqp/AmqpReceiverOptions.java

```diff
@@ -115,50 +113,9 @@ public AmqpReceiverOptions setQos(String qos) {
     return this;
   }
 
-  /**
-   * Custom desired capabilities.
-   * A registry of commonly defined source capabilities and their meanings is maintained at
-   * <a href="http://www.amqp.org/specification/1.0/source-capabilities">AMQP Source Capabilities</a>.
-   *
-   * @return the list of capabilities, empty if none.
-   */
-  public List<String> getDesiredCapabilities() {
-    if (desiredCapabilities == null) {
-      return new ArrayList<>();
-    }
-    return desiredCapabilities;
-  }
-
-  /**
-   * Sets the desired capabilities.
-   * A registry of commonly defined source capabilities and their meanings is maintained at
-   * <a href="http://www.amqp.org/specification/1.0/source-capabilities">AMQP Source Capabilities</a>.
-   *
-   * @param desiredCapabilities the list of desired capabilities.
-   * @return the options.
-   */
-  public AmqpReceiverOptions setDesiredCapabilities(List<String> desiredCapabilities) {
-    this.desiredCapabilities = desiredCapabilities;
-    return this;
-  }
-
-  /**
-   * Adds a desired capability.
-   *
-   * @param capability the capability to be added, must not be {@code null}.
-   * @return the options
-   */
-  public AmqpReceiverOptions addDesiredCapability(String capability) {
-    Objects.requireNonNull(capability, "The capability must not be null");
-    if (this.desiredCapabilities == null) {
-      this.desiredCapabilities = new ArrayList<>();
-    }
-    this.desiredCapabilities.add(capability);
-    return this;
-  }
 
   /**
-   * Gets the list of offered capabilities.
+   * Gets the list of offered capabilities for the source.
```

Comment:
These aren't offered capabilities, they are more of an expectation for capabilities in this case. The peer has definitive say on what the capabilities of the Source are in the case of a client consuming (and similarly for the Target in case of a client sending).

Same applies to the 'setCapabilities' method below, not 'offered'.